test/integration/features/basic.feature

```diff
@@ -56,104 +50,84 @@ Feature: Basic
   Scenario: User can login to the server
-    When executing "oc login --username=developer --password=developer"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc login --username=developer --password=developer" succeeds
+    Then stdout should contain
      """
      Login successful
      """
   
   Scenario: User can create new namespace ruby for application ruby-ex
-    When executing "oc new-project ruby"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc new-project ruby" succeeds
+    Then stdout should contain
      """
      Now using project "ruby"
      """
   
   Scenario: User can deploy application ruby-ex to namespace ruby 
-    When executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" succeeds
+    Then stdout should contain
      """
      Success
      """
-    When executing "oc rollout status deploymentconfig ruby-ex --watch"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc rollout status deploymentconfig ruby-ex --watch" succeeds
+    Then stdout should contain
      """
      "ruby-ex-1" successfully rolled out
      """
   
   Scenario: User can create route for ruby-ex to make it visiable outside of the cluster
-    When executing "oc expose svc/ruby-ex"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc expose svc/ruby-ex" succeeds
+    Then stdout should contain
      """
      exposed
      """
   
   Scenario: User can delete namespace ruby
-    When executing "oc delete project ruby"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc delete project ruby" succeeds
+    Then stdout should contain
      """
      "ruby" deleted
      """
   
   Scenario: User can log out the session
-    When executing "oc logout"
-    Then stderr should be empty
-     And exitcode should equal 0
-     And stdout should contain
+    When executing "oc logout" succeeds
+    Then stdout should contain
      """
      Logged "developer" out
      """
   # End of Ruby application ruby-ex deployment
   
   Scenario: Stopping Minishift
     Given Minishift has state "Running"
-     When executing "minishift stop"
+     When executing "minishift stop" succeeds
      Then Minishift should have state "Stopped"
 
   Scenario: Deleting Minishift
     Given Minishift has state "Stopped"
-     When executing "minishift delete"
+     When executing "minishift delete" succeeds
      Then Minishift should have state "Does Not Exist"
```

Comment:
What's about adding more more tests after deletion, something like 'minishift ip'. This will actually return a non 0 exit code. This way we can keep a test for the case where we don't use the "succeeds" step. Not so much about testing Minishift, but more to show that this step also works.